app/src/main/java/com/example/hepiplant/PlantAddActivity.java

```diff
@@ -142,8 +142,7 @@ private void postRequestPlant(){
                     @Override
                     public void onResponse(JSONObject response) {
                         onPostResponsePlant(response);
-                        Intent intent = new Intent(getApplicationContext(),PlantsListActivity.class);
-                        startActivity(intent);
+
```

Comment:
u mnie tu jest finish(), któraś by musiała tu pododawać

Reply:
sory na masterze add jest już zmieniony z finish(), jedynie edit i delete plant nie są

Reply:
To poprawię to przy okazji w edit i delete :)

Reply:
wtedy musi być w klasie poprzedniej (w szczegółach rośliny) resume z nowym pobraniem, zrobiłam tak w poście i sales offer

Reply:
Okej, zrobię to analogicznie jak ty